<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1185</width>
    <height>705</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <spacer name="rightHorizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <spacer name="leftHorizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <spacer name="bottomVerticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
      <item>
       <widget class="Visualization3DWidget" name="openGLWidget"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="firstTab">
           <attribute name="title">
            <string>1</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="secondTab">
           <attribute name="title">
            <string>2</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="thirdTab">
           <attribute name="title">
            <string>3</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="fourthTab">
           <attribute name="title">
            <string>4</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="fifthTab">
           <attribute name="title">
            <string>5</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="sixthTab">
           <attribute name="title">
            <string>6</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="seventhTab">
           <attribute name="title">
            <string>7</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="eighthTab">
           <attribute name="title">
            <string>8</string>
           </attribute>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="selectFunctionComboBox"/>
        </item>
        <item>
         <widget class="QCheckBox" name="gridVisibility">
          <property name="text">
           <string>Grid Visibility</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="axisVisibility">
          <property name="text">
           <string>Axis Visibility</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="returnButton">
          <property name="text">
           <string>Return to initial state</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <spacer name="upperVerticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1185</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Visualization3DWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>visualization_3d_widget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
